contrib/qubes/src/qubes_fwupdmgr.py

```diff
@@ -128,7 +128,7 @@ def refresh_metadata(self, whonix=False, metadata_url=None):
             raise Exception("Manual metadata refresh failed!!!")
 
     def _get_dom0_updates(self):
-        """Gathers infromations about available updates."""
+        """Gathers information about available updates."""
```

Comment:
codespell made me do it.

(I personally prefer this spelling, but Google suggested the simpler fix and I didn't feel like being particularly opinionated.)

Reply:
I think information is better here.